.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "search.useIgnoreFiles": false,
+  "search.exclude": {}
+}
```

Comment:
Let's add this file to .gitignore

Reply:
I altered the search parameters earlier to find some definitions in the node modules, but yes it belongs in gitignore - fixing it now